Kickstarter-iOS/Views/Controllers/PledgePaymentMethodsViewController.swift

```diff
@@ -99,12 +99,20 @@ final class PledgePaymentMethodsViewController: UIViewController {
 
         self.messageDisplayingDelegate?.pledgeViewController(self, didErrorWith: errorMessage)
       }
+
+    self.applePayButton.rac.hidden = self.viewModel.outputs.applePayButtonHidden
   }
 
   // MARK: - Configuration
 
-  func configureWith(_ user: User) {
-    self.viewModel.inputs.configureWith(user)
+  func configureWith(_ value: (User, Project)) {
```

Comment:
Can this be `func configure(with value: (user: User, project: Project)) {`, reads a bit better than `value.0`